src/Traits/BinaryFlags.php

```diff
@@ -196,4 +201,120 @@ public function checkAnyFlag($mask)
     {
         return $this->checkFlag($mask, false);
     }
+
+    /**
+     * Return the current element
+     *
+     * @link  https://php.net/manual/en/iterator.current.php
+     * @return mixed Can return any type.
+     * @since 5.0.0
+     */
+    public function current()
+    {
+        return $this->mask & $this->currentPos;
```

Comment:
I think it would be more logical to return the name of the flag as `current()` and the `bitmask` as `key()`